woocommerce/woocommerce-android
# Fix Clear filters button visibility on Order list

### Please do not merge this until iOS devs confirm the tracking event is not added to iOS yet (if it is we need to sync on the name)

Closes: #8340 


### Description


Currently, the `clear button` for the selected filters on the order list screen blinks and disappears. This PR fixes this issue and add the tracking event to match the `clear button` on the Product filters screen

### Testing instructions


1. Go to Orders
2. click on filters
3. select one or two filters
4. make sure the clear button is visible at all times after selecting the filter
5. click on clear and make sure they are cleared

### Images/gif


<img src="https://user-images.githubusercontent.com/30724184/219051777-8db690d3-92b8-43f1-a07f-74f1427ece51.gif" width="350"/>
